src/components/Repeat/RepeatWords/RepeatWords.tsx

```diff
@@ -29,11 +35,12 @@ const RepeatWords: React.FC<RepeatWordsProps> = ({ daysRepeat }) => {
         <S.InputStyle
           $fontColorLabel='purpleDark'
           $isLightTeam={true}
-          {...inputNameElement('word_Transate', 'wordTransate', 'Word Transate')}
+          {...inputNameElement('word_Translate', 'wordTranslate', 'Word Translate')}
           onChange={(value: string) => setWordTranslate(value)}
           value={wordTranslate}
           required
           $isCorrect={isCorrect}
+          disabled={wordsRepeat[currentWordIndex]?.wordUserAnswer !== undefined}
```

Comment:
jesli w tym pliku używasz: `wordsRepeat` tylko w tym jednym miejscu to lepiej tutaj juz przekazac końcową wartośc tego sprawdzenia. czyli 
`{isDisabled}`
a cały ten warunek `wordsRepeat[currentWordIndex]?.wordUserAnswer !== undefined` zrobić w rodzicu. nie musisz wtedy przekazywac tyle rzeczy do dziecka